src/azure-cli/azure/cli/command_modules/botservice/_params.py

```diff
@@ -60,10 +60,9 @@ def load_arguments(self, _):
         c.argument('description', options_list=['--description', '-d'], help='The description of the bot.', arg_group='Registration Bot Specific')
         c.argument('endpoint', options_list=['-e', '--endpoint'], help='The messaging endpoint of the bot.', arg_group='Registration Bot Specific')
         c.argument('msa_app_id', options_list=['--appid'], help='The Microsoft account ID (MSA ID) to be used with the bot.')
-        c.argument('password', options_list=['-p', '--password'], help='The Microsoft account (MSA) password for the bot. Used to authorize messages being sent to the bot.')
+        c.argument('password', options_list=['-p', '--password'], help='The Microsoft account (MSA) password for the bot. Used to authorize messages being sent to the bot. Required for Web App Bots.', arg_group='Web App Bot Specific')
         c.argument('tags', arg_type=tags_type)
-        c.argument('language', options_list=['--lang'], arg_type=get_enum_type(SUPPORTED_LANGUAGES), help='The language to be used to create the bot.', arg_group='Web Bot Specific')
-        c.argument('version', options_list=['-v', '--version'], help='The Microsoft Bot Builder SDK version to be used to create the bot', arg_type=get_enum_type(['v4']), arg_group='Web Bot Specific')
+        c.argument('language', options_list=['--lang'], arg_type=get_enum_type(SUPPORTED_LANGUAGES), help='The language to be used to create the bot.', arg_group='Web App Bot Specific')
```

Comment:
Could you please use options_list=['--language', '-l']?

Reply:
`'-l'` would conflict with the global `'--location', '-l'` I think...

I could update it to be `options_list=['--language', '--lang']`, or should I leave it as is?

Reply:
After checking the code, `-l` would conflict with the `location` parameter

Reply:
It will be better to provide one full parameter name for language. What about ['--language', '-a']? ['--language','---lang'] is also acceptable.

Reply:
I prefer  ['--language','---lang'] as 'a' is not intuit.

Reply:
@Juliehzl and @jsntcy I'm open to `['--language', '--lang']`, is this acceptable?